tests/trilinos/vector_reinit_to_linear_map.mpirun=2.output

```diff
@@ -0,0 +1,83 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2004 - 2018 by the deal.II authors
```

Comment:
I meant that this should read
```
// Copyright (C) 2004 - 2018 by the deal.II authors	
[...]
```

Reply:
Not sure I understand what change to make here. Do you mean there should be nothing below the copyright line except the description of the test?

Reply:
No, what @masterleinad was pointing out is that your new test has copyright years 2004-2018, which is not technically correct. Since you're adding this test this year, the date range should only include 2018.